spec/features/ingredients/index_spec.rb

```diff
@@ -0,0 +1,31 @@
+require "rails_helper"
+
+RSpec.describe "Ingredient Index Page", type: :feature do
+  before :each do
+    @beef_wellington = Recipe.create!(name: 'Beef Wellington', complexity: 9, genre: "French")
+    @short_ribs = Recipe.create!(name: 'Braised Short Ribs', complexity: 5, genre: "Spanish")
+    @tiramisu = Recipe.create!(name: 'Tiramisu', complexity: 10, genre: "Italian")
+
+    @salt = @beef_wellington.ingredients.create!(name: 'Salt', cost: 5)
+    @ground = @beef_wellington.ingredients.create!(name: 'Ground Beef', cost: 30)
+    @pastry = @beef_wellington.ingredients.create!(name: 'Puff Pastry', cost: 20)
+
+    @meat = @short_ribs.ingredients.create!(name: 'Short Ribs', cost: 50)
+    @wine = @short_ribs.ingredients.create!(name: 'Red Wine', cost: 35)
+    @onion = @short_ribs.ingredients.create!(name: 'White Onion', cost: 5)
+
+    @coffee = @tiramisu.ingredients.create!(name: 'Coffee Grounds', cost: 15)
+    @ice_cream = @tiramisu.ingredients.create!(name: 'Italian Ice Cream', cost: 20)
+    @batter = @tiramisu.ingredients.create!(name: 'Cake Batter', cost: 17)
```

Comment:
Really great object setup for this test! I would say though, just make sure you don't spend too much time creating objects up front and then get distracted/tire yourself out before actually writing _the test_ -- this will come into play more for the mid-mod and finals, which are timed synchronously. 
For this test for example, I would imagine 2-3 objects overall being sufficient. 
However, the nice thing is now, you could copy-paste this data into `seeds.rb` or your other test files for speedier setup.